### ISSUE TITLE
Export partition status

### ISSUE THREAD

**Issue by ercliou** [jirwin/burrow_exporter#11]
I'd like to export the status of each partition too.
We can always write some logic at prometheus end, but Burrow already does this well.
https://github.com/linkedin/Burrow/wiki/http-request-consumer-group-status
These are the valid status strings: NOTFOUND, OK, WARN, ERR, STOP, STALL

Edit:
We shall model them as separate time series
```
kafka_burrow_partition_state{cluster="MY_CLUSTER",group="MY_GROUP",partition="13",topic="MY_TOPIC",state:"OK"} 1
kafka_burrow_partition_state{cluster="MY_CLUSTER",group="MY_GROUP",partition="13",topic="MY_TOPIC",state:"STOP"} 1
```
something like https://www.robustperception.io/exposing-the-software-version-to-prometheus/

**Comment by kanga333** [comment]
@ercliou 
Hello.
I also want this metrics.
You seem to have made some changes after forking, but are you planning to send a patch upstream?

**Comment by ercliou** [comment]
I ended up implementing by sending all metrics at every scrap. When the status is not the matched one, it sends `0`. This increases 1:5 with number of partitions (could be a problem if you have a lot of them).
e.g.
```
kafka_burrow_partition_state{cluster="MY_CLUSTER",group="MY_GROUP",partition="13",topic="MY_TOPIC",state:"OK"} 1
kafka_burrow_partition_state{cluster="MY_CLUSTER",group="MY_GROUP",partition="13",topic="MY_TOPIC",state:"STOP"} 0
kafka_burrow_partition_state{cluster="MY_CLUSTER",group="MY_GROUP",partition="13",topic="MY_TOPIC",state:"REWIND"} 0
kafka_burrow_partition_state{cluster="MY_CLUSTER",group="MY_GROUP",partition="13",topic="MY_TOPIC",state:"STALL"} 0
kafka_burrow_partition_state{cluster="MY_CLUSTER",group="MY_GROUP",partition="13",topic="MY_TOPIC",state:"WARN"} 0
```
I could send a patch if @jirwin agrees with this :)

**Comment by jirwin** [comment]
I'm +1 to this. Partition count isn't generally unbound. Maybe it could be enabled by a command line flag, so people can use their own judgement as to whether the surge in new time series is acceptable to them. Maybe `--per-partition-stats` or something?

**Comment by jirwin** [comment]
Fixed by #19.
